### ISSUE TITLE
Missing include_dirs.js?

### ISSUE THREAD

**Issue by tyler-johnson** [rvagg/nan#262]
Hello, just updated nan to 1.5.2 but now I am getting a `Error: Cannot find module 'nan'` when I try to require it. I think this is because the `include_dirs.js` file is missing in the NPM tarball.

**Comment by rvagg** [comment]
Confirmed:

```
$ curl -s https://registry.npmjs.org/nan/-/nan-1.5.2.tgz | tar ztvf -| grep include_dirs.js
```

We'll get a 1.5.3 out to deal with this.

Workaround for now is to use 1.6.0:

```
$ curl -s https://registry.npmjs.org/nan/-/nan-1.6.0.tgz | tar ztvf -| grep include_dirs.js
-rw-rw-r-- 1000/1000        55 2014-11-07 05:43 package/include_dirs.js
```

**Comment by kkoopa** [comment]
Weird, don't see how that went missing all of a sudden... I just did npm publish as usual.

**Comment by rvagg** [comment]
yeah, it's got me scratching my head too, very odd

**Comment by kkoopa** [comment]
Can you have a crack at republishing it and see if it was a random fluke?

**Comment by rvagg** [comment]
Made a v1.5 branch and did a 1.5.3:

```
$ npm pack && tar ztvf nan-1.5.*.tgz | grep include_dirs.js
nan-1.5.3.tgz
-rw-r--r-- 1000/1000        55 2014-07-03 16:16 package/include_dirs.js
```

published now:

```
$ curl -s https://registry.npmjs.org/nan/-/nan-1.5.3.tgz | tar ztvf -| grep include_dirs.js
-rw-r--r-- 1000/1000        55 2014-07-03 16:16 package/include_dirs.js
```

_\*shrug\*_

**Comment by kkoopa** [comment]
Strange indeed.

**Comment by tyler-johnson** [comment]
Thanks!

**Comment by kkoopa** [comment]
But now 1.5.3 is the newest, so we need a 1.6.1

**Comment by rvagg** [comment]
haha, doh! completely forgot about tags

**Comment by rvagg** [comment]
published a 1.6.1 but have found out that we don't need to publish to change tags: https://twitter.com/othiym23/status/558427463823654914:
> npm tag <package>@<version> latest
